docs/docs/gallery.md

```diff
@@ -0,0 +1,7 @@
+# Description 
```

Comment:
Maybe change the title to "Simulation gallery" or something like that?

Reply:
Done.